game.py

```diff
@@ -0,0 +1,62 @@
+
+# Object for things that happen in the game
+# name is what the event is, length is how many turns it lasts (1 and on), effect is how it affects the player (damages the player, initiates battle), description is the text describing the event
+class Event:
+    def __init__(self, name, length, effect, description):
+        self.name = name
+        self.length = length
+        self.effect = effect
+        self.description = description
+
+# Effects: poison (10 damage per turn), 
+    effects = {
```

Comment:
You can actually initiate the effects as a state of the object, so that different instances of this object could have different effects if you wanted! Or you could just hard code the effects similar to how you did here. Check out my suggestion below!